nss-day-cohort-60/rare-client-glorious-gitastrophies
# Author naming convention

### To test:

1. Launch app and login
2. Navigate to User Management tab
3. Check and see if list of users populates
4. If so, click on user and verify that proper information renders
